main/src/library/Boxable.flix

```diff
@@ -239,9 +239,28 @@ instance Boxable[(a, b, c)] with Boxable[a], Boxable[b], Boxable[c] {
     }
 }
 
-// TODO: Benj: Add for Tuples 3, 4, 5.
+instance Boxable[(a, b, c, d)] with Boxable[a], Boxable[b], Boxable[c], Boxable[d] {
```

Comment:
Why require the inner types to be Boxable? The boxing is shallow. Shouldn't we require ToString and Order instead?